app/services/data_migrations/delete_work_history_audits.rb

```diff
@@ -0,0 +1,29 @@
+module DataMigrations
+  class DeleteWorkHistoryAudits
+    TIMESTAMP = 20240917154444
+    MANUAL_RUN = true
+    BATCH_SIZE = 1500
+
+    def change
+      time_now = Time.zone.now
+      counter = 1
+
+      relation.in_batches(of: BATCH_SIZE) do |batch|
+        next_batch_time = time_now + (counter * 5).seconds
```

Comment:
I don't think we need to spread these out over time - the operations will be very quick as they don't instantiate ActiveRecord models